app/src/app/api/v0/city/[city]/file/route.ts

```diff
@@ -60,130 +59,119 @@ export const GET = apiHandler(async (_req: Request, context) => {
   return NextResponse.json({ data: userFilesTransformed });
 });
 
-export const POST = apiHandler(async (req: NextRequest, context) => {
-  const userId = context.session?.user.id;
-  const service = new NotificationService();
-  const user = context.session?.user;
-  const cityId = context.params.city;
-
-  const city = await db.models.City.findOne({
-    where: {
-      cityId,
-    },
-  });
-
-  if (!city) {
-    throw new createHttpError.Unauthorized("Unauthorized");
-  }
+export const POST = apiHandler(
+  async (req: NextRequest, { params, session }) => {
+    const service = new NotificationService(); // TODO cache this/ make it a singleton
```

Comment:
We can create a ticket for this

Reply:
I'll make it part of the PR feedback ticket we've added to refactor this function